### PR TITLE
imx-test, imx-sw-pdm

### PR DESCRIPTION
**Pull request by thochstein**

### CODE REVIEW

**Review by otavio — APPROVED** [review]

**Comment by otavio** [comment]
What sw-pdm is?

**Comment by thochstein** [comment]
> What sw-pdm is?

@aalonso, can you give more detail?

File: recipes-multimedia/imx-sw-pdm/imx-sw-pdm_1.0.1.bb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 NXP Semiconductors
+
+DESCRIPTION = "NXP PDM to PCM Software Decimation SIMD Library"
+LICENSE = "Proprietary"
```
Comment (otavio) [inline recipes-multimedia/imx-sw-pdm/imx-sw-pdm_1.0.1.bb:4]: The current description would fit for a summary. Please add a description to this.

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-multimedia/imx-sw-pdm/imx-sw-pdm_1.0.1.bb (hunk 1 above)
Comment (otavio) [reply recipes-multimedia/imx-sw-pdm/imx-sw-pdm_1.0.1.bb:4]: Please add a more detailed description as the current one does not explain what it provides.

Comment (thochstein) [reply recipes-multimedia/imx-sw-pdm/imx-sw-pdm_1.0.1.bb:4]: @aalonso, can you help me here?

**Review by otavio — APPROVED** [review]

**Pull request merged**